src/model.jl

```diff
@@ -14,11 +14,12 @@ mutable struct PowerNetwork
     gmax
     A
     B
+    F
```

Comment:
Can you add doc for `PowerNetwork` clarifying all the variables? Would be useful for anyone first bumping into the code